_opensearch/rest-api/_script-apis/index.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: default
+title: Script APIs
+parent: REST API reference
+has_children: true
+nav_order: 160
+redirect_from:
+  - /opensearch/rest-api/script-apis/
+---
+
+# Script APIs
+
+The script APIs allow you to work with stored scripts. Stored scripts are part of the cluster state and reduce compilation time and enhance search speed. The default scripting language is Painless. 
+
+You perform the following operations on stored scripts:
+* create
```

Comment:
Can we make these links to pages?

Reply:
@kolchfa-aws A list of links to pages is automatically generated.

Reply:
I see the above list rendering as plain text. I suggest we make those links to the pages they refer to.

Reply:
@Naarcha-AWS, I looked at the production version and the links in the bulleted list (see attachment) are broken. They each include path segment `script-apis` that needs to be changed to `_script-apis`.

![opensearch-script-link-errors](https://user-images.githubusercontent.com/30219968/188733534-244fc785-1c95-467e-9ee3-6b2132e39068.png)